docs/source/mooncake-store-api/python-binding.md

```diff
@@ -0,0 +1,866 @@
+# Mooncake Store Python API
+
```

Comment:
It seems this file covers a wide range of content,  which is far more than `python api`. Will it be better if we name it as `Mooncake Store Usage` or similar names?

Reply:
The majority of the content revolves around the Python API, so I think retain the same name is preferable.